src/qibo_cloud_backends/qibo_client.py

```diff
@@ -46,4 +46,5 @@ def execute_circuit(self, circuit, initial_state=None, nshots=1000):
                 NotImplementedError,
                 "The use of an `initial_state` is not supported yet.",
             )
-        return self.client.run_circuit(circuit, nshots=nshots, device=self.platform)
+        job = self.client.run_circuit(circuit, nshots=nshots, device=self.platform)
+        return job.result(verbose=0)
```

Comment:
Could it be useful to allow the user for setting the desired `verbosity`, maybe not through the `execute_circuit` directly, but rather globally for the backend with a property for example:
``` python
@property
def verbosity(self):
    return self.verbosity

@property.setter
def verbosity(self, verb: int):
    self.verbosity = verb
```

Reply:
Yes, that is possible but probably we should add this argument at the constructor level so `set_backend` does the proper configuration setup.